samples/hello-world/HW-qio.ipynb

```diff
@@ -0,0 +1,265 @@
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The resulting list should contain a list of target names, along with some details for each one. A _target_ is a remote solver that can perform optimization or quantum computation. In this example, we will focus on optimization."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### ❕ Do you see `microsoft.simulatedannealing-parameterfree.cpu` in your list of targets? If so, you're ready to keep going.\r\n",
+        "\r\n",
+        "Don't see it? You may need to add Microsoft QIO to your workspace to run this sample. Navigate to the **Providers** page in the portal and click **+Add** to add the Microsoft QIO provider."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Microsoft QIO: The optimization provider\r\n",
+        "Azure Quantum connects with first- and third-party optimizers to deliver optimization solutions. These offerings are called *providers*. Each provider can offer multiple *targets* with different capabilities. See the table below for Microsoft QIO's targets.\r\n",
+        "\r\n",
+        "**Microsoft QIO target ID anatomy**: `microsoft.{algorithm name}.{hardware type}`. Algorithm names ending in `-parameterfree` use automatic parameter selection. \r\n",
+        "\r\n",
+        "Optimization Algorithm | Target IDs | Python Class\r\n",
+        "| --- | ---| ---| --- | -- |\r\n",
+        "[Simulated annealing](https://docs.microsoft.com/azure/quantum/optimization-simulated-annealing) | `microsoft.simulatedannealing.cpu`,<br/>`microsoft.simulatedannealing-parameterfree.cpu`,<br/>`microsoft.simulatedannealing.fpga` |  `azure.quantum.optimization.SimulatedAnnealing` |\r\n",
+        "[Parallel tempering](https://docs.microsoft.com/azure/quantum/optimization-parallel-tempering) | `microsoft.paralleltempering.cpu`,<br/>`microsoft.paralleltempering-parameterfree.cpu`,<br/>`microsoft.paralleltempering.fpga`  | `azure.quantum.optimization.ParallelTempering`\r\n",
+        "[Tabu search](https://docs.microsoft.com/en-us/azure/quantum/optimization-tabu-search) | `microsoft.tabu.cpu`,<br/>`microsoft.tabu-parameterfree.cpu`  | `azure.quantum.optimization.Tabu`\r\n",
+        "[Quantum Monte Carlo](https://docs.microsoft.com/azure/quantum/optimization-quantum-monte-carlo) | `microsoft.qmc.cpu`  | `azure.quantum.optimization.QuantumMonteCarlo`\r\n",
+        "[Population annealing](https://docs.microsoft.com/azure/quantum/optimization-population-annealing) | `microsoft.populationannealing.cpu`  | `azure.quantum.optimization.PopulationAnnealing`\r\n",
```

Comment:
"`azure.quantum.optimization.PopulationAnnealing`" can be removed, and "`microsoft.populationannealing-parameterfree.cpu`" should be added